iocore/eventsystem/IOBuffer.cc

```diff
@@ -135,11 +135,21 @@ MIOBuffer::write_and_transfer_left_over_space(IOBufferReader *r, int64_t alen, i
 #endif
 
 int64_t
-MIOBuffer::write(IOBufferReader *r, int64_t alen, int64_t offset)
+MIOBuffer::write(IOBufferReader *r, int64_t len, int64_t offset)
```

Comment:
Because there are now two methods that do this sort of writing, that has been refactored in to another method which is called from the two standard entry points of writing a block chain.

Reply:
you mean `buffer chain`. I keep reading it as block chain.

Reply:
Buffer block chain. We had those before those parvenus did.